index.html

```diff
@@ -13,6 +13,1026 @@
+                </li>
+                <li class="c3">
+                  <button
+                    type="button"
+                    class="c5"
+                    aria-label="Save articles and visit later on any device"
+                  >
+                    <svg
+                      stroke="none"
+                      width="16px"
+                      height="25px"
+                      viewBox="0 0 16 25"
+                      class="c5-1"
+                    >
+                      <g
+                        stroke="none"
+                        stroke-width="1"
+                        fill="none"
+                        fill-rule="evenodd"
+                      >
+                        <g stroke="#666" fill="none">
+                          <path
+                            fill="none"
+                            d="M0.5,0.5 L0.5,24.5 L8.1716264,19.5137775 L15.5,24.4404006 L15.5,0.5 L0.5,0.5 Z"
+                          ></path>
+                        </g>
+                      </g>
+                    </svg>
+                  </button>
+                </li>
+                <li class="c6">
+                  <button
+                    type="button"
+                    class="c6-1"
+                    aria-label="615 comments. Click to read and post comments."
+                    id="comments-speech-bubble-top"
+                  >
+                    <i class="c6-2">
+                      <span class="OpenCommentsButton-text--2uwLE">615 </span>
+                    </i>
+                  </button>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="article">
+        <p class="par1">
+          CAMBRIDGE, Mass. — One night late in 1979, an itinerant young
+          physicist named Alan Guth, with a new son and a year’s appointment at
+          Stanford, stayed up late with his notebook and equations, venturing
+          far beyond the world of known physics.
+        </p>
+        <p class="par2">
+          <span
+            >He was trying to understand why there was no trace of some exotic
+            particles
+          </span>
+          <span id="fade1">
+            that should have been created in the Big Bang. Instead he discovered
+            what
+          </span>
+          <span id="fade2">
+            might have made the universe bang to begin with. A potential hitch
+            in the
+          </span>
+          <span id="fade3">
+            presumed course of cosmic evolution could have infused space itself
+            with a
+          </span>
+          <span id="fade4">
+            special energy that exerted a repulsive force, causing the universe
+            to swell faster
+          </span>
+          <span id="fade5">
+            than the speed of light for a prodigiously violent instant.</span
+          >
+        </p>
+      </div>
+      <div class="editPicks">
```

Comment:
use some absolute positioning to div down, currently it's overlapping the image.

Reply:
Positioning fixed and code review request submitted.